src/main/java/net/dv8tion/jda/api/audio/AudioReceiveHandler.java

```diff
@@ -86,4 +88,26 @@
      *         The user audio data
      */
     void handleUserAudio(UserAudio userAudio);
+
+    /**
+     * This method is a filter predicate used by JDA to determine whether or not to include a
+     * {@link net.dv8tion.jda.api.entities.User User}'s audio when creating a CombinedAudio packet.
+     * <p>
+     * This method is especially useful in creating whitelist / blacklist functionality for receiving audio.
+     * <p>
+     * A few possible examples:
+     *  <li>Have this method always return false for Users that are bots.</li>
```

Comment:
Isn't this missing a `<ul>` somewhere?

Reply:
Oh. Yeah, you're right.
I guess the javadoc renderer just understood the lack of `<ul>` when I previewed it.